common/logger.go

```diff
@@ -32,7 +44,7 @@ func CreateLogger(logFile string) *Logger {
 	writer := io.MultiWriter(os.Stdout, f)
 	handler := slog.NewTextHandler(writer, &slog.HandlerOptions{
 		AddSource: true,
-		Level:     slog.LevelDebug,
+		Level:     slog.LevelInfo,
```

Comment:
maybe also getting this in the constructor, and using Info as default if no level provided?

Reply:
A slight problem, there isn't a real optional parameter in go. Will look into it

Reply:
Probably overloading functions
```go
func CreateLogger(logPath string, logFile string, level slog.Level) *Logger {
return CreateLogger(logPath, logFile)
}

func CreateLogger(logPath string, logFile string, level slog.level) *Logger {
...
}
```

Other than that I don't really have an idea

Reply:
I think I would take a more go-like approach and go with and Option struct 

```golang

type LoggerOptions {
  LogPath string
  LogFile string
  Level   slog.Level (or int or something)
}

func CreateLogger(options *LoggerOptions)...
```

then check the struct and fill default values if needed